Math/circumference_of_Circle.py

```diff
@@ -0,0 +1,11 @@
+# Defining the function to calculate the circumference of a circle
+def Circumference():
+    radius = float(input('Enter Radius Of Circle: '))
+
+    # The formula to calculate circumference
+    circumference = 2*3.14159*radius
```

Comment:
It is better to use PI from the math standard library

Reply:
Also, it would be more readable if you add space between operators.
eg: 2 * PI * radius

Reply:
I avoided using the maths library to make it as default Pythonic as possible, as some people don't want to use certain libraries when there is another way. The number of digits of PI i used in is accurate for quite a bit, even NASA only uses 15 digits.

Secondly, i will definitely fix the code to make it more readible. Is the maths library is a must, also let me know

Reply:
It was just a recommendation, however, make a static variable for PI

Reply:
I updated the value of PI to make it static